### ISSUE TITLE
Feature: tooling-wide Config file or ENV variables

### ISSUE THREAD

**Issue by jackorp** [pvalena/theprototype#2]
TODO:
  - [ ] create-pr.sh fails if USERNAME is unknown.

_ _ _ _

Originally:

_Right now there is a default username set in `create-pr.sh`._
_Instead of auto supplying USERNAME if the arg is missing, the script could exit with a message._
_How about some shell "config file" with these common values that would be sourced?_

**Comment by pvalena** [comment]
Yes, the `Config file` is a long planned feature. It shouldn't however be mandatory.

Let's use this issue to track the feature and its requirements in respective scripts.
